test/e2e/performanceprofile/functests/6_mustgather_testing/mustgather.go

```diff
@@ -109,5 +109,5 @@ func checkfilesExist(listOfFiles []string) error {
 		}
 		Expect(info.Size()).ToNot(BeZero())
 	}
-	return nil 
+	return nil
```

Comment:
this whole commit should be squashed in the previous

Reply:
done